apps/node-fastify/src/test/test-utils.ts

```diff
@@ -0,0 +1,50 @@
+import { OrbSync } from 'orb-sync-lib';
+
+/**
+ * Deletes test data from the database to ensure clean test state.
+ *
+ * @param orbSync - The OrbSync instance
+ * @param table - The table name to delete from
+ * @param ids - Array of IDs to delete
+ */
+export async function deleteTestData(orbSync: OrbSync, table: string, ids: string[]) {
+  if (!ids.length) return;
+  const postgresClient = orbSync.getPostgresClient();
+  const placeholders = ids.map((_, index) => `$${index + 1}`).join(',');
+  await postgresClient.query(`DELETE FROM orb.${table} WHERE id IN (${placeholders})`, ids);
+}
+
+/**
+ * Fetches invoices from the database by their IDs.
+ *
+ * @param orbSync - The OrbSync instance
+ * @param ids - The IDs of the invoices to fetch
+ * @returns The invoices
+ */
+export async function fetchInvoices(orbSync: OrbSync, ids: string[]) {
```

Comment:
If you rename the `ids` parameter to invoice ids - you can omit the entire comment, as it describes itself

Reply:
Maybe rename to `fetchInvoicesFromDatabase` to make it 100% clear